test/check-application

```diff
@@ -1425,6 +1425,24 @@ class TestApplication(testlib.MachineCase):
         b.click('#containers-images tbody tr:contains("alpine:latest") td.pf-c-table__toggle button')
         b.wait_in_text("#containers-images tbody tr:contains('alpine:latest') td[data-label='Used by']", 'unused')
 
+        # Create a container without starting it
+        container_name = "busybox-not-started"
+        b.wait_visible('#containers-images td[data-label="Image"]:contains("busybox:latest")')
+        b.click('#containers-images tbody tr:contains("busybox:latest") .ct-container-create')
+        b.wait_visible('div.pf-c-modal-box header:contains("Create container")')
+
+        b.wait_in_text("#run-image-dialog-image", "busybox:latest")
+        b.set_input_text("#run-image-dialog-name", container_name)
+
+        # Uncheck start container after creation
+        b.click("#start-after-creation")
+        b.click('.pf-c-modal-box__footer button:contains("Create")')
+        b.wait_not_present("div.pf-c-modal-box")
+
+        b.wait_not_in_text("#containers-containers", container_name)
```

Comment:
This is race-y. It waits for container not to be in the list - which might be just true by default (tests are fast so it might not yet be present), then changes to all containers and then waits for it to be present. It unfortunately does not say anything whether the container is running or not.

It should be safe to just switch to `all` and wait for the container status to be `created`.

Reply:
Looking at the test of the tests, it might be better to use `waitContainer`